app/lib/katello/resources/registry.rb

```diff
@@ -33,10 +33,10 @@ def load_class
               registry_url = uri.to_s
 
               # Assume the registry uses the same CA as the Smart Proxy
-              ca_cert_file = Setting[:ssl_ca_file]
+              ca_cert_file = ::Cert::Certs.backend_ca_cert_file(:pulp3)
             elsif (container_config = SETTINGS.dig(:katello, :container_image_registry))
               registry_url = container_config[:crane_url]
-              ca_cert_file = container_config[:registry_ca_cert_file]
```

Comment:
I debated this change. It suggests nobody uses this since the [installer sets `crane_ca_cert_file`](https://github.com/theforeman/puppet-katello/blob/2e15151a6d324149071c4e240bed78b5d977adf6/templates/katello.yaml.erb#L42)